### PR TITLE
Make sure the changelog notification dropdown is only shown once

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4434

dismissable-message

### CODE REVIEW

File: components/changelog/ChangelogTrigger.js [hunk 1]
```diff
@@ -88,8 +89,10 @@ const ChangelogTrigger = props => {
             backgroundSize={backgroundSize}
             url="/static/images/flame-red.svg"
           />
-          <ChangelogNotificationDropdown />
-        </Dropdown>
+          <DismissibleMessage messageId="ChangelogTrigger.dropdownNotification">
```
Comment (Betree) [inline components/changelog/ChangelogTrigger.js:92]: Would be nice to put this message in https://github.com/opencollective/opencollective-frontend/blob/b657bf1fd79e8ce3a260163b6001dc9071c169c8/lib/constants/dismissable-help-message.js#L4

**Review by Betree — APPROVED** [review]
Nice & simple as it should be, good job. I hope this will make a better UX

File: components/changelog/ChangelogTrigger.js (hunk 1 above)
Comment (SudharakaP) [reply components/changelog/ChangelogTrigger.js]: Ops, I missed that there was a constants file. Just added that there. 😄

**Comment by SudharakaP** [comment]
> Nice & simple as it should be, good job. I hope this will make a better UX

Thanks a bunch. 😄

**Pull request merged**